$NetBSD: patch-generate_unix_Makefile.config,v 1.3 2024/08/04 10:52:06 nia Exp $

Remove -Werror.
Avoid using GCC-specific flags. fixes build with clang

--- generate/unix/Makefile.config.orig	2016-09-30 16:43:55.000000000 +0000
+++ generate/unix/Makefile.config
@@ -185,7 +185,6 @@ CWARNINGFLAGS = \
     -Wall\
     -Wbad-function-cast\
     -Wdeclaration-after-statement\
-    -Werror\
     -Wformat=2\
     -Wmissing-declarations\
     -Wmissing-prototypes\
@@ -206,7 +205,6 @@ CWARNINGFLAGS += \
     -Wmissing-declarations\
     -Wmissing-field-initializers\
     -Wnested-externs\
-    -Wold-style-definition\
     -Wno-format-nonliteral\
     -Wredundant-decls
 #
@@ -218,10 +216,6 @@ ifneq ($(HOST), _FreeBSD)
 
     ifneq ($(HOST), _APPLE)
         CWARNINGFLAGS += \
-            -Woverride-init\
-            -Wlogical-op\
-            -Wmissing-parameter-type\
-            -Wold-style-declaration\
             -Wtype-limits
     endif
 endif
